index.html

```diff
@@ -0,0 +1,983 @@
+                            <a><img alt="" src="images/opinions/leida-crisostomo.png"/></a>
+                            <h3>Florida Woman Robs Mailman with Plastic Gun</h3>
+                        </article>
+                        <article><h6>TEST</h6>
+                            <a><img alt="" src="images/opinions/01-gettyimages-1074721578.jpg"/></a>
+                            <h3>Con Edison Transformer Causes Laguardia To Close</h3>
+                        </article>
+                        <article><h6>TEST</h6>
+                            <a><img alt="" src="images/opinions/01-gettyimages-1074721578.jpg"/></a>
+                            <h3>Americaís Oldest Living World War II Veteran Dies</h3>
+                        </article>
+                        <article><h6>TEST</h6>
+                            <a><img alt="" src="images/opinions/01-gettyimages-1074721578.jpg"/></a>
+                            <h3>Tribune Warns Spectrum Subscribers They Could Lose NFL</h3>
+                        </article>
+                    </section>
+                    <section class="signUpBox">
+                        <div class="content">
+                            <div class="titleFlex">
+                                <h3>
+                                    <span>
+                                        <svg xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" enable-background="new 0 0 29 28" version="1.1" viewBox="0 0 29 28" xml:space="preserve">
+                                            <image width="29" height="28" display="none" overflow="visible" xlink:href="../N.png"></image>
+                                            <polygon class="st0" points="8 22 12 22 12 15 17 22 21 22 21 6 17 6 17 13 12 6 8 6"/>
+                                        </svg>
+                                    </span>
+                                    SIGN UP FOR OUR NEWSLETTER
+                                </h3>
+                                <a class="signUp">SIGN UP</a>
+                                <a class="updatePerformances">Update your preferences ª</a>
+                            </div>
+
+                        </div>
+                    </section>
+                </div>
+            </div>
+        </section>
+        <section class="container-fluid inTheMagazine">
+            <div class="row">
+                <div class="colone col-md-3">
+                    <div class="seeAllFeatures">
+                        <div class="image">
+                            <img alt="" src="images/seeAllFeatures/20180104-cover.png">
+                            <span class="description">JANUARY 04 ISSUE</span>
+                        </div>
+                        <a>See All Features</a>
+                    </div>
+                </div>
+                <div class="colone col-md-9">
+                    <div class="row">
+                        <div class="col-md-12">
```

Comment:
You can also have the col without size and if it's one column it takes the whole width

Reply:
I have made changes... Thanks...